flake8diff/vcs/git.py

```diff
@@ -65,3 +69,7 @@ def changed_files(self):
         return filter(self.filter_file,
                       iter(_execute(' '.join(command))
                            .splitlines()))
+
+    def _is_git_repository(self):
+        return _execute(
+            '{vcs} status'.format(vcs=self.vcs), strict=True, log_errors=False)
```

Comment:
No need to change, but we could also do this with a check for a directory named `.git` in all parent directories from the current directory.  Something like:

```python
from os import getcwd
from os.path import sep, join, exists

def _is_git_repository(self):
    cwd = getcwd()
    return any(map(lambda i: exists(join(join(sep, *cwd.split(sep)[:i]), '.git')),
                   range(len(cwd.split(sep)), 0, -1)))
```